prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua

```diff
@@ -13,6 +13,7 @@
 {{ $JWT_ALLOW_EMPTY := .Env.JWT_ALLOW_EMPTY | default "0" | toBool -}}
 {{ $JWT_AUTH_TYPE := .Env.JWT_AUTH_TYPE | default "token" -}}
 {{ $JWT_ENABLE_DOMAIN_VERIFICATION := .Env.JWT_ENABLE_DOMAIN_VERIFICATION | default "false" | toBool -}}
+{{ $JWT_SIGN_TYPE := .Env.JWT_SIGN_TYPE | default "RS512" -}}
```

Comment:
This will break existing setups. Default to null so the module picks the right default, you can still choose RS512 is you want to.